src/modules/sync/sync/webSpotlight.ts

```diff
@@ -1,14 +1,28 @@
 import { ManagementClient } from "@kontent-ai/management-sdk";
 import { match, P } from "ts-pattern";
 
+import { logInfo, LogOptions } from "../../../log.js";
 import { WebSpotlightDiffModel } from "../types/diffModel.js";
 
-export const updateWebSpotlight = (client: ManagementClient, diffModel: WebSpotlightDiffModel): Promise<unknown> =>
+export const updateWebSpotlight = (
+  client: ManagementClient,
+  diffModel: WebSpotlightDiffModel,
+  logOptions: LogOptions,
+): Promise<unknown> =>
   match(diffModel)
-    .with({ change: "none" }, () => Promise.resolve())
-    .with({ change: "deactivate" }, () => client.deactivateWebSpotlight().toPromise())
+    .with({ change: "none" }, () => {
+      logInfo(logOptions, "standard", "No web spotlight to update");
```

Comment:
This sounds like the environment doesn't have web spotlight enabled. Something like "No web spotlight changes to perform" would be better.